packages/backend/migrations/20241203112902_add_tenant_to_wallet_address.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return Promise.all([
+    knex.schema.alterTable('walletAddresses', function (table) {
+      table.uuid('tenantId').index().notNullable()
+      //table.foreign(['tenantId']).references('tenants.id')
+    })
+  ])
```

Comment:
What is the intention with the commented out line here? Waiting on something? I imagine this would be a foreign key and we could uncomment this.

Reply:
Yip, missed that one, thanks @BlairCurrey fixed.